File: TwoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int ant = -1;
        int act = -1;

        for (int i = 0; i < nums.length; i++) {
            for (int j= 0; j < i; j++) {
                if (nums[i] + nums[j] == target) {
                    act = i;
                    ant = j;
                }
            }
        }
        return new int[]{ant, act};
    }
}

File: TwoSum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        first = -1
        second = 1
        while first == -1 or nums[first] + nums[second] != target or first == second:
            first += 1 
            for num in nums:
                if nums[first] + num == target and first != nums.index(num):
                    second = nums.index(num)

        return [first, second]
